tasks/parse-pdr/index.js

```diff
@@ -68,3 +80,13 @@ function handler(event, context, callback) {
   cumulusMessageAdapter.runCumulusTask(parsePdr, event, context, callback);
 }
 exports.handler = handler;
+
+// use node index.js local to invoke this
+justLocalRun(() => {
+  //const payload = require('@cumulus/test-data/cumulus_messages/parse-pdr.json'); // eslint-disable-line global-require, max-len
+  const payload = require('/Users/pcappela/Development/gitc-deploy/ingest-sips/messages/parse-pdr.json');
+  handler(payload, {}, (e, r) => {
+    //console.log(e);
+    console.log(JSON.stringify(r, null, '\t'));
```

Comment:
Add ` // eslint-disable-line no-console` to the end of this line to make eslint happier.